docs/api/covidcast-signals/doctor-visits.md

```diff
@@ -33,15 +33,20 @@ percentage of COVID-related doctor's visits in a given location, on a given day.
 Note that because doctor's visits may be reported to the health system partners
 several days after they occur, these signals are typically available with
 several days of lag. This means that estimates for a specific day are only
-available several days later.
+available several days later. 
 
 The amount of lag in reporting can vary, and not all visits are reported with
 the same lag. After we first report estimates for a specific date, further data
 may arrive about outpatients visits on that date. When this occurs, we issue new
-estimates for those dates. This means that a reported estimate for, say, June
+estimates for those dates to backfill any missing data. This means that a reported estimate for, say, June
 10th may first be available in the API on June 14th and subsequently revised on
 June 16th.
 
+ As insurnace claims are available at a significant and variable latency, the signal experiences heavy backfill with data delayed for a couple of weeks. 
+ Note that we expect estimates available for the most recent X days to be substantially revised due to later data revisions, estimates for past dates are more static due to these revisions having already occured. 
```

Comment:
```suggestion
We expect estimates available for the most recent X days to change substantially in later data revisions. Estimates for dates more than W days in the past are expected to remain fairly static, as most major revisions have already occurred.
```

Reply:
I pulled 7 days from the blog post ("For a given date of service, we typically receive only about half of the total claims within seven days following that date.") and extended it to 14 to hedge against the arrival of bulk of the second half of the claims. I estimated 28 days based on the issue size of the doctor-visits signal, which is six weeks. If we estimate no measurable changes after six weeks, 4 weeks is probably safe to classify as "fairly static"